### ISSUE TITLE
Blank screen with the new opencore instructions.

### ISSUE THREAD

**Issue by piyuyi** [hieplpvip/ASUS-ZENBOOK-UX410-HACKINTOSH#87]
**What is your laptop?**
Asus zenbook UX430UAR
**Which version of macOS are you using?**
10.15.7
**Describe the problems**
I have followed the new instructions for the creation of the opencore version (selecting kaby lake R in my case) but when I boot from usb EFI, it goes black and restarts itself.

Can you fix it please?
Thank you.

**Comment by hieplpvip** [comment]
Hi, please boot with boot-args `-v keepsyms=1 debug=0x100` and give me the log.

**Comment by piyuyi** [comment]
I have done it and the same thing happens. The problem is that it does not enter to opencore, a black screen appears directly and reboots.

**Comment by hieplpvip** [comment]
@Ubsefor Can you boot with latest build?

**Comment by Ubsefor** [comment]
> @Ubsefor Can you boot with latest build?

Working for me rn, OC 0.6.2

**Comment by hieplpvip** [comment]
@piyuyi Do you see any USB UEFI entry in BIOS boot menu?

**Comment by Ubsefor** [comment]
> @piyuyi Do you see any USB UEFI entry in BIOS boot menu?

I’m using it right now for what I can tell (no diffs in config etc., for exception of AppleIntelBacklight )

**Comment by hieplpvip** [comment]
@piyuyi @Ubsefor Please send me your EFI folder. Thanks.

**Comment by Ubsefor** [comment]
Oh, I’m sorry, didn’t see new commits in OC repo
Will test it at Friday, now I’m busy AF :(

**Comment by piyuyi** [comment]
> @hieplpvip Do you see any USB UEFI entry in BIOS boot menu?
Yes, i can select the EFI USB from the boot menu, and after this it shows directly the black screen (no opencore menus).

> @hieplpvip Please send me your EFI folder. 
[EFI.zip](https://github.com/hieplpvip/Asus-Zenbook-Hackintosh/files/5381321/EFI.zip)

I think the problem is in some parameter of the config.plist, because I have moved your config.plist to another EFI folder working, and the same thing happens, it does not get to enter the opencore menu.

Thank you ;-)

**Comment by hieplpvip** [comment]
@piyuyi Try setting `SecureBootModel` to `Disabled`

**Comment by teekiai** [comment]
@hieplpvip Same problem on me. ux430uar - with disabled SecureBootModel - no diferences.
Not entering to OC menu

**Comment by piyuyi** [comment]
> @piyuyi Try setting `SecureBootModel` to `Disabled`

+1, no lucky....

**Comment by hieplpvip** [comment]
Please use DEBUG version of OpenCore. Remember to enable debugging log as shown [here](https://dortania.github.io/OpenCore-Install-Guide/troubleshooting/debug.html). Thanks a lot.

**Comment by piyuyi** [comment]
> Please use DEBUG version of OpenCore. Remember to enable debugging log as shown [here](https://dortania.github.io/OpenCore-Install-Guide/troubleshooting/debug.html). Thanks a lot.

Here it is:
[opencore-2020-10-15-154444.txt](https://github.com/hieplpvip/Asus-Zenbook-Hackintosh/files/5386210/opencore-2020-10-15-154444.txt)
.

Thank you.

**Comment by hieplpvip** [comment]
OpenCore seems to load successfully. Please boot with argument `-v keepsyms=1 debug=0x100`

**Comment by piyuyi** [comment]
> OpenCore seems to load successfully. Please boot with argument `-v keepsyms=1 debug=0x100`

The log debug that I sent you, is already executed with those arguments ... :-(

**Comment by hieplpvip** [comment]
Please set `ShowPicker` to `true`, `Timeout` to `5`, and check the boot menu.

**Comment by piyuyi** [comment]
> Please set `ShowPicker` to `true`, `Timeout` to `5`, and check the boot menu.

HEY, NOW IT WORKS!!!!...... 👏👏👏

Thank you very much for your time and your patience, I hope I don't bother you anymore 🤗.

**Comment by hieplpvip** [comment]
Is this the first time you boot macOS? As I remember, on first boot (or after NVRAM reset), OpenCore will not boot to macOS automatically. Instead, it will boot into Windows and reboot. Took me a lot of time to figure it out.

**Comment by piyuyi** [comment]
> Is this the first time you boot macOS? As I remember, on first boot (or after NVRAM reset), OpenCore will not boot to macOS automatically. Instead, it will boot into Windows and reboot. Took me a lot of time to figure it out.

OK but no it´s my first boot on mac, I've been using your clover version perfectly for a long time.
I have bad news, the broadcom wifi has stopped working with your opencore version... any suggestions?

**Comment by Ubsefor** [comment]
> > Is this the first time you boot macOS? As I remember, on first boot (or after NVRAM reset), OpenCore will not boot to macOS automatically. Instead, it will boot into Windows and reboot. Took me a lot of time to figure it out.
> 
> OK but no it´s my first boot on mac, I've been using your clover version perfectly for a long time.
> I have bad news, the broadcom wifi has stopped working with your opencore version... any suggestions?

You need to disable `AirportBrcmFixup.kext/Contents/PlugIns/AirPortBrcm4360_Injector.kext` in kernel section, it is being deprecated for what I can tell. Doesn’t work in Big Sur anyway, so try to set `MaxKernel` to some version lower than the one you’re using.

Check the official repo for **BrcmPatchRam** and **OpenCore Install Guide,** the most issues are covered there – just read carefully and check your setup.

**Comment by piyuyi** [comment]
> > > Is this the first time you boot macOS? As I remember, on first boot (or after NVRAM reset), OpenCore will not boot to macOS automatically. Instead, it will boot into Windows and reboot. Took me a lot of time to figure it out.
> > 
> > 
> > OK but no it´s my first boot on mac, I've been using your clover version perfectly for a long time.
> > I have bad news, the broadcom wifi has stopped working with your opencore version... any suggestions?
> 
> You need to disable `AirportBrcmFixup.kext/Contents/PlugIns/AirPortBrcm4360_Injector.kext` in kernel section, it is being deprecated for what I can tell. Doesn’t work in Big Sur anyway, so try to set `MaxKernel` to some version lower than the one you’re using.
> 
> Check the official repo for **BrcmPatchRam** and **OpenCore Install Guide,** the most issues are covered there – just read carefully and check your setup.

OK, i will try.

Thank you.

**Comment by hieplpvip** [comment]
Should be fixed in https://github.com/hieplpvip/Asus-Zenbook-Hackintosh/commit/4082d0d6ed274cfed48fef3e3f940261f0df98cc

**Comment by teekiai** [comment]
> Should be fixed in [4082d0d](https://github.com/hieplpvip/Asus-Zenbook-Hackintosh/commit/4082d0d6ed274cfed48fef3e3f940261f0df98cc)

Everything is working on UX430 (Kaby Lake-R) not just on 10.15.7. Trying 11.0 beta 9 works everything like on 10.15.7
Only audio jack on my laptop is cracking i'l try old method with jack fix when I got time.

Thank You for making everything in one place to my Hackintosh 👍
![BigSur](https://user-images.githubusercontent.com/4189686/96454969-3d82bc00-1225-11eb-9ff9-aa6c59a87420.jpg)

**Comment by shichem420** [comment]
> Please set `ShowPicker` to `true`, `Timeout` to `5`, and check the boot menu.

I'm having the same issue, in what section do I change this?
